Add beforeConnect / beforeSubscribe docs (#752)
ParsePlatform/Docs: _includes/cloudcode/cloud-code.md

# LiveQuery Triggers

## beforeConnect

*Available only on parse-server cloud code starting 4.3.0*

> You can run custom Cloud Code before a user attempts to connect to your LiveQuery server. You can do this with the `beforeConnect` method. For instance, this can be used to only allow users that have logged in to connect to the LiveQuery server. 

Reviewer: ```suggestion
You can run custom Cloud Code before a user attempts to connect to your LiveQuery server with the `beforeConnect` method. For instance, this can be used to only allow users that have logged in to connect to the LiveQuery server. 
```

```javascript
Parse.Cloud.beforeConnect(request => {
  if (!request.user) {
    throw "Please login before you attempt to connect."
  }
});
```

> In most cases, the `connect` event called the first time the client calls `subscribe`. If this is your use case, you can listen for errors using this event.

Reviewer: ```suggestion
In most cases, the `connect` event is called the first time the client calls `subscribe`. If this is your use case, you can listen for errors using this event.
```

```javascript
Parse.LiveQuery.on('error', (error) => {
  console.log(error);
});
```

## beforeSubscribe

*Available only on parse-server cloud code starting 4.3.0*

> In some cases you may want to transform the incoming subcription query, adding an additional limit or increasing the default limit, adding extra includes or restrict the results to a subset of keys. You can do so with the `beforeSubscribe` trigger. 

Reviewer: ```suggestion
In some cases you may want to transform the incoming subscription query. Examples include adding an additional limit, increasing the default limit, adding extra includes or restricting the results to a subset of keys. You can do so with the `beforeSubscribe` trigger. 
```
